workshop-resources/keyboard-navigation/event-reviews/2023-07-10-pydatasphinx.md

```diff
@@ -0,0 +1,285 @@
+
+### General notes
+
+This is the place for anything a reviewer thinks is important that doesn't belong below. This could be something out of scope of the review, something that will impact all other tests, or the like.
+
+Notes:
+
+### [Home page](https://pydata-sphinx-theme.readthedocs.io/en/latest/index.html) review
+
+#### Content order
+
+When accessed via keyboard, the content order is logical. ([WCAG 2.2 Focus order](https://www.w3.org/TR/WCAG22/#focus-order))
+
+1. From the top of the page, press the Tab key repeatedly until reviewer reaches the end of content.
+2. Reviewers will know it is the end because if they press Tab once more their keyboard focus will return to the browser.
+3. If the reviewer cannot make it to the end of content, please take note of where the focus gets stuck.
+
+Does the content order follow reviewer expectations based on reading the content?
+Result:
+
+Does the content order make sense for interacting with the content?
+Result:
+
+Are any major content areas missed when navigating via keyboard?
+Result:
+
+Are any interactive content areas missed when navigating via keyboard?
+Result:
+
+Other notes or recommendations:
+
+#### Areas to navigate
+
+All areas of the tool can be accessed using keyboard navigation. ([WCAG 2.2 Keyboard](https://www.w3.org/TR/WCAG22/#keyboard))
+
+1. Navigate via the Tab key major content areas
+2. Use the arrow keys to navigate within major content areas.
+
+| Area | Able to navigate to the area (yes/no/not applicable) | Able to navigate fully within the area (yes/no/not applicable) | Able to navigate out of the area (yes/no/not applicable) |
+|---|---|---|---|
+| Banner | a | b | c |
+| Main menu bar | d | e | f |
+| Main content area | g | h | i |
+| Main area | j | k | l |
+| Bottom of page navigation | m | n | o |
+| Footer | p | q | r |
+| Other (add as needed) | s | t | u |
+
+Other notes by area:
+
+#### Keyboard/tab traps
+
+When navigating via keyboard, there are no areas where keyboard focus can enter but not exit. The focus never gets "trapped." ([WCAG 2.2 No keyboard trap](https://www.w3.org/TR/WCAG22/#no-keyboard-trap))
+
+1. From the top of the page, press the Tab key repeatedly until reviewer reaches the end of content.
+2. Reviewers will know it is the end because if they press Tab once more their keyboard focus will return to the browser.
+3. If the reviewer cannot make it to the end of content, please take note of where the focus gets stuck. If the reviewer can make it to all content, please note.
+4. Optional: complete steps 1–3 over any areas that may have been missed by the content order. Using the mouse to move focus to an area and trying to tab out may be helpful for these cases.
+
+Keyboard/tab traps:
+- tab trap 1
+- tab trap 2
+
+Other notes or recommendations:
+
```

Comment:
```suggestion

[@gabalafou]: I did not find any tab traps. I checked several different pages: [home](https://pydata-sphinx-theme.readthedocs.io/en/latest/index.html), [kitchen sink](https://pydata-sphinx-theme.readthedocs.io/en/latest/examples/kitchen-sink/index.html), [PyData library styles](https://pydata-sphinx-theme.readthedocs.io/en/latest/examples/pydata.html), [execution libraries](https://pydata-sphinx-theme.readthedocs.io/en/latest/examples/execution.html), and more.

I doubt there are any tab traps on this site because it is not a very interactive site. 
```